backend/gsr_booking/admin.py

```diff
@@ -9,13 +14,14 @@ class GroupMembershipInline(admin.TabularInline):
 
     readonly_fields = ["name"]
 
-    def name(self, obj):
-        return obj.user.get_full_name()
+    def name(self, obj: GroupMembership) -> str:
+        user = cast(UserType, obj.user)
```

Comment:
does this force cast the object? what is the difference between this and `user: UserType = obj.user`?